commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/Drive.scala

```diff
@@ -9,10 +9,18 @@ trait Drive {
   type DriveSignal
   type DriveVelocity
 
+  type DrivetrainHardware
```

Comment:
I don't quite understand why we are having these as a type aliases and not as members